packages/core/src/modules/event/event-committee-service.ts

```diff
@@ -0,0 +1,19 @@
+import { Committee, Event } from "@dotkomonline/types"
+import { EventCommitteeRepositoryImpl } from "./event-committee-repository"
+
+export interface EventCommitteeService {
+  getCommitteesForEvent(eventId: Event["id"]): Promise<Committee[]>
+}
+
+export class EventCommitteeServiceImpl implements EventCommitteeService {
+  constructor(private readonly committeeOrganizerRepository: EventCommitteeRepositoryImpl) {}
+
+  async getCommitteesForEvent(eventId: Event["id"]): Promise<Committee[]> {
+    const committees = await this.committeeOrganizerRepository.getAllCommittees(eventId)
+    return committees
+  }
+
+  async setEventCommittees(eventId: Event["id"], organizers: Committee["id"][]): Promise<void> {
```

Comment:
This can stay as "set" but we should do the difference logic here as explained in the repository.